addons/binding/org.openhab.binding.feed.transform/OSGI-INF/feedTransformationService.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+  Copyright (c) 2010-2017 by the respective copyright holders.
+ 
+  All rights reserved. This program and the accompanying materials
+  are made available under the terms of the Eclipse Public License v1.0
+  which accompanies this distribution, and is available at
+  http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="FeedTransformationService">
```

Comment:
Should the name `FeedTransformationService` start with a lowercase, I do not know?